go/protocol/chat1/local.go

```diff
@@ -1365,13 +1365,15 @@ type GetThreadQuery struct {
 
 type GetThreadLocalRes struct {
 	Thread           ThreadView                    `codec:"thread" json:"thread"`
+	Offline          bool                          `codec:"offline" json:"offline"`
 	RateLimits       []RateLimit                   `codec:"rateLimits" json:"rateLimits"`
 	IdentifyFailures []keybase1.TLFIdentifyFailure `codec:"identifyFailures" json:"identifyFailures"`
 }
 
 type GetInboxLocalRes struct {
 	ConversationsUnverified []Conversation                `codec:"conversationsUnverified" json:"conversationsUnverified"`
 	Pagination              *Pagination                   `codec:"pagination,omitempty" json:"pagination,omitempty"`
+	Offline                 bool                          `codec:"offline" json:"offline"`
```

Comment:
Any chance this will break old versions? I thought we've had sigs fail to verify for adding new fields?